commands/inventory/add_agent_mongodb_exporter.go

```diff
@@ -48,33 +52,60 @@ func (res *addAgentMongodbExporterResult) String() string {
 }
 
 type addAgentMongodbExporterCommand struct {
-	PMMAgentID          string
-	ServiceID           string
-	Username            string
-	Password            string
-	CustomLabels        string
-	SkipConnectionCheck bool
-	TLS                 bool
-	TLSSkipVerify       bool
-	PushMetrics         bool
+	PMMAgentID                    string
+	ServiceID                     string
+	Username                      string
+	Password                      string
+	CustomLabels                  string
+	SkipConnectionCheck           bool
+	TLS                           bool
+	TLSSkipVerify                 bool
+	TLSCertificateKey             string
```

Comment:
we don't need this as a separate field. it can be just variable.
`TLSCa` too

Reply:
*File removed from here and used as a variable.